content/source/docs/enterprise/api/workspaces.html.md

```diff
@@ -306,41 +274,50 @@ $ curl \
-- `terraform_version` (`string: <optional>`) - Specifices the version of Terraform to use for this workspace.
-- `working-directory` (`string: ''`) - Specifies the directory that Terraform will execute within. This defaults to the root of your configuration and is typically set to a subdirectory matching the environment when multiple environments exist within the same configuration.
-- `vcs-repo.branch` (`string: ''`) - Specifies the repository branch that Terraform will execute from. If left null or as an empty string, this defaults to the repository's default branch (e.g. `master`) .
-- `vcs-repo.oauth-token-id` (`string: <optional>`) - Specifies the VCS Connection (OAuth Conection + Token) to use as identified. This ID can be obtained from the [oauth-tokens](./oauth-tokens.html) endpoint.
-- `vcs-repo.ingress-submodules` (`boolean: false`) - Specifies whether submodules should be fetched when cloning the VCS repository.
-- `vcs-repo.identifier` (`string: <required>`) - This is the reference to your VCS repository in the format :org/:repo
+| Key path                                      | Type    | Default    | Description                                                                                                                                                                                                                             |
+| --------------------------------------------- | ------- | ---------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `data.type`                                   | string  |            | Must be `"workspaces"`.                                                                                                                                                                                                                 |
+| `data.attributes.name`                        | string  |            | Specifies the name of the workspace, which can only include letters, numbers, `-`, and `_`. This will be used as an identifier and must be unique in the organization.                                                                  |
+| `data.attributes.auto-apply`                  | boolean | false      | Whether to automatically apply changes when a Terraform plan is successful.                                                                                                                                                             |
+| `data.attributes.terraform-version`           | string  |            | The version of Terraform to use for this workspace.                                                                                                                                                                                     |
```

Comment:
(same comments above re defaults, requirements, and legal values for some of these.)